src/jsapi.rs

```diff
@@ -4490,14 +4490,14 @@ extern "C" {
                                                                                              scopeChain:
                                                                                                  *mut AutoObjectVector)
      -> *mut JSObject;
-    fn _Z25JS_BufferIsCompilableUnitP9JSContextN2JS6HandleIP8JSObjectEEPKcj(cx:
+    fn _Z25JS_BufferIsCompilableUnitP9JSContextN2JS6HandleIP8JSObjectEEPKcm(cx:
                                                                                 *mut JSContext,
                                                                             obj:
                                                                                 Handle<*mut JSObject>,
                                                                             utf8:
                                                                                 *const i8,
                                                                             length:
-                                                                                u32)
+                                                                                u64)
```

Comment:
>
> This is size_t. Should we be be using usize?
>
Probably, yes. It seems like bindgen generated wrong code for size_t? I
wonder if that afflicts more sites than just this one.